src/main/waveform/waveform.mixins.js

```diff
@@ -349,6 +367,37 @@ define(['konva'], function (Konva) {
     };
   }
 
+  /**
+   * Returns a zero padded string for the given number
+   *
+   * @param {Number} number to be padded
+   * @param {Number} length of the final string
+   * @param {Boolean} if true or omitted, the number will be left-padded
```

Comment:
YUIDoc does not pull anything from the JavaScript so you must list the attribute names - `{Number} number number to be padded`, `{Boolean} prepend if...`, etc.